packages/create-graphback/src/init/communityTemplates.ts

```diff
@@ -19,7 +19,7 @@ export const externalTemplates = [
     repos: [{
       uri: 'https://github.com/namit-chandwani/nestjs-apollo-mongodb-backend',
       branch: 'master',
-      path: '/'
+      path: '.'
```

Comment:
```suggestion
```

Reply:
I tried doing that initially, but received an error. The reason being that `path` has been declared as a compulsory field in the TemplateRepository interface (as shown below).

https://github.com/aerogear/graphback/blob/a45d0c5b0202127da1dd84d5dd2a12a09c247276/packages/create-graphback/src/init/templateMetadata.ts#L15-L24

So `path: '.'` was the only thing that seemed to work.